.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/index.js":"1","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/reportWebVitals.js":"2","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/App.js":"3","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/components/Main.jsx":"4","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Styled-components/MainStyle.js":"5","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Assets/Device/Device.jsx":"6","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/components/HeaderMenu.jsx":"7","/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Styled-components/HeaderMenuStyled.jsx":"8"},{"size":478,"mtime":1609753391204,"results":"9","hashOfConfig":"10"},{"size":364,"mtime":1609753391204,"results":"11","hashOfConfig":"10"},{"size":204,"mtime":1609753391204,"results":"12","hashOfConfig":"10"},{"size":5464,"mtime":1609753391204,"results":"13","hashOfConfig":"10"},{"size":2543,"mtime":1609753391204,"results":"14","hashOfConfig":"10"},{"size":377,"mtime":1609579525565,"results":"15","hashOfConfig":"10"},{"size":536,"mtime":1609753391204,"results":"16","hashOfConfig":"10"},{"size":594,"mtime":1609753509425,"results":"17","hashOfConfig":"10"},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"os9lny",{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"24","messages":"25","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"26","messages":"27","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"28","messages":"29","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"30"},{"filePath":"31","messages":"32","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"33","messages":"34","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/index.js",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/reportWebVitals.js",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/App.js",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/components/Main.jsx",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Styled-components/MainStyle.js",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Assets/Device/Device.jsx",[],["35","36"],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/components/HeaderMenu.jsx",[],"/home/fxc/Documents/P3/lille-0920-js-p3-favart/src/Styled-components/HeaderMenuStyled.jsx",[],{"ruleId":"37","replacedBy":"38"},{"ruleId":"39","replacedBy":"40"},"no-native-reassign",["41"],"no-negated-in-lhs",["42"],"no-global-assign","no-unsafe-negation"]
```

Comment:
Oh, now FX steals Thomas' place in this file :p 
(Add it to the `.gitignore`, damnit ! >_<)